tests/integration/tests/test_smoke.py

```diff
@@ -17,3 +17,25 @@ def test_smoke(session_instance: harness.Instance):
     config = result.stdout.decode()
     assert len(config) > 0
     assert "server: https://192.168.210.41" in config
+
```

Comment:
looks like we would be better using a separate instance in `test_smoke`. That way we can keep the bootstrap config (the extra configs) closer to the test code here. Because right now, they are very far away, and it is not obvious how the bootstrap config reaches this place.

Reply:
done